celery/backends/redis.py

```diff
@@ -340,3 +341,22 @@ def db(self):
     @deprecated.Property(4.0, 5.0)
     def password(self):
         return self.connparams['password']
+
+
+class SentinelBackend(RedisBackend):
+    """Sentinel task result store."""
+
+    def __init__(self, *args, **kwargs):
+        super(SentinelBackend, self).__init__(*args, **kwargs)
+
+        _get = self.app.conf.get
+
+        self.broker_url = _get('BROKER_URL') or {}
```

Comment:
Hi, George
This code sample is intended when BROKER_URL == BACKEND_URL.
I will agree that this is not flexible, but it is fixable. It does not matter.
This can be redesigned to receive exactly BACKEND_URL.
The important thing is that the celery bakend still does not support the sentinel.
I also wrote that I would be grateful if someone would rewrite this code more correctly, but so far I have not seen a significant contribution and therefore I am confident,
I also wrote that I would be grateful if someone would rewrite this code more correctly, but so far I have not seen a significant contribution.